javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,112 @@
+const usersName = [];
```

Comment:
I would say there is only one username, so I would suggest 
```suggestion
let username = "";
```